UserApi/UserApi/Helper/UserProfileHelper.cs

```diff
@@ -25,6 +25,12 @@ public class UserProfileHelper
             {"Civil Money Claims", AdGroup.MoneyClaims}
         };
 
+        private static readonly Dictionary<AdGroup, string> CaseTypeMappings = new Dictionary<AdGroup, string>
+        {
+            { AdGroup.MoneyClaims, "Civil Money Claims" },
```

Comment:
Extract constants ?

Reply:
They're only used here so I'd rather inline them to keep the line count down.